llama-index-core/llama_index/core/callbacks/base.py

```diff
@@ -139,6 +139,10 @@ def set_handlers(self, handlers: List[BaseCallbackHandler]) -> None:
         """Set handlers as the only handlers on the callback manager."""
         self.handlers = handlers
 
+    @classmethod
+    def __modify_schema__(cls, schema: Dict[str, Any]) -> None:
+        schema.update(type="object", default={})
+

```

Comment:
what was the motivation for this again? a comment would help.

Reply:
avoids pydantic serialization issues I think. Will add a note